From c7dc8e23d0646200424179c16ccac2961df8da86 Mon Sep 17 00:00:00 2001
From: mossywell <mark@mossywell.com>
Date: Fri, 28 Mar 2014 07:26:49 +0000
Subject: [PATCH 22/24] platinum: Add audio/x-flac MIME type to Neptune

Change streaming FLAC MIME type from application/octet to audio/x-flac to
support Frontier Silicon systems (such as the Axis X3) that require
this. (Others don't care.)

FLAC not added to DefaultDlnaMap as * suffices.

(Commit amend: tabs --> spaces)
(Commit amend: flac --> x-flac)
---
 lib/libUPnP/Neptune/Source/Core/NptHttp.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/libUPnP/Neptune/Source/Core/NptHttp.cpp b/lib/libUPnP/Neptune/Source/Core/NptHttp.cpp
index 1700afc..8c5d587 100644
--- a/lib/libUPnP/Neptune/Source/Core/NptHttp.cpp
+++ b/lib/libUPnP/Neptune/Source/Core/NptHttp.cpp
@@ -2779,6 +2779,7 @@ NPT_HttpFileRequestHandler_DefaultFileTypeMap[] = {
     {"aif",  "audio/x-aiff"},
     {"aifc", "audio/x-aiff"},
     {"aiff", "audio/x-aiff"},
+    {"flac", "audio/x-flac"},
     {"mka",  "audio/x-matroska"},
     {"mpa",  "audio/mpeg"},
     {"mp2",  "audio/mpeg"},
-- 
1.7.11.msysgit.0

